index.js

```diff
@@ -3650,26 +3779,30 @@
   //. See also [`elem`](#elem).
   //.
   //. ```javascript
-  //. > S.find(n => n < 0, [1, -2, 3, -4, 5])
-  //. Just(-2)
+  //. > S.find (S.lt (0)) ([1, -2, 3, -4, 5])
+  //. Just (-2)
   //.
-  //. > S.find(n => n < 0, [1, 2, 3, 4, 5])
+  //. > S.find (S.lt (0)) ([1, 2, 3, 4, 5])
   //. Nothing
   //. ```
-  function find(pred, xs) {
-    return Z.reduce(
-      function(m, x) { return m.isJust ? m : pred(x) ? Just(x) : Nothing; },
-      Nothing,
-      xs
-    );
+  function find(pred) {
+    return function(xs) {
+      return Z.reduce (
+        function(m, x) {
+          return m.isJust ? m : pred (x) ? Just (x) : Nothing;
+        },
+        Nothing,
+        xs
+      );
+    };
   }
   _.find = {
     consts: {f: [Z.Foldable]},
-    types: [$.Predicate(a), f(a), $Maybe(a)],
+    types: [$.Predicate (a), f (a), $Maybe (a)],
     impl: find
   };
 
-  //# unfoldr :: (b -> Maybe (Pair a b)) -> b -> Array a
+  //# unfoldr :: (b -> Maybe (Array2 a b)) -> b -> Array a
```

Comment:
We're now using [`$.Array2`][1], which replaced `$.Pair`.


[1]: https://github.com/sanctuary-js/sanctuary-def#Array2